ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -455,6 +456,50 @@ private void updateSubviewClipStatus(View subview) {
     }
   }
 
+  @Override
+  public boolean getChildVisibleRect(View child, Rect r, android.graphics.Point offset) {
```

Comment:
I think you ought to override the method flavor that _has_ the `forceParentCheck` param and take it into account. It's pretty easy to add, isn't it?

Reply:
@d4vidi
I wanted to, but since the original for that flavor has `@hide` in the comment, it's not available to override.  I tried implementing it anyway, but when the ViewGroup implementation tries to call it (via a cast) it never reaches the subclass implementation anyway :(

Reply:
See https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/view/ViewGroup.java#6185

Reply:
Apparently, the ViewGroup implementation of getChildVisibleRect() is the only one that calls into an auxiliary function to force recursion when not necessary. It seems it was written for some kind of internal testing, and then marked `@hide` to prevent it from being used by the public.